fixtures/ios_app_with_frameworks/.gitignore

```diff
@@ -0,0 +1 @@
+Derived/
```

Comment:
Do we want `.gitignore` at this level?

Reply:
I'd say so. That way we can run the test assuming there are no files in that directory.